modules/Functions.js

```diff
@@ -0,0 +1,9 @@
+import { timeAndDate } from './variables.js';
+import { DateTime } from './luxon.js';
+
+export default function dateAndTimeNow() {
+  setInterval(() => {
+    const date = DateTime.now().toLocaleString(DateTime.DATETIME_FULL_WITH_SECONDS);
+    timeAndDate.innerHTML = date;
+  }, 0);
```

Comment:
- [ ] according to the requirements we are required to use ES6 syntax which includes arrow functions instead of the `function`  keyword

Reply:
Fixed